docs/notebooks/jax_Array.md

```diff
@@ -0,0 +1,858 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "PxHrg4Cjuapm"}
+
+# `jax.Array`
+
++++ {"id": "pFtQjv4SzHRj"}
+
+**The features required by `jax.Array` are not supported by the Colab TPU runtime at this time.**
```

Comment:
I might add a note here to orient the reader, something like:
```
This tutorial discusses parallelism via `jax.Array`, the unified array object model available in JAX vX.X.X and newer.
```

Reply:
Done.